### ISSUE TITLE
WS2M not accessible through AG community

### ISSUE THREAD

**Issue by danielreispereira** [ropensci/nasapower#32]
The code below works, but if I change WS10M for WS2M, it returns an error:

> ""Error: 
> You have entered an invalid value for `community` for the supplied `pars`. One or more `pars` are not, available for `AG`, please check.""

```
get_power(community = "AG",
                         lonlat = c(-49.554,-20.862, -48.562, -19.851),
                         pars = c("PRECTOT", "T2M_MAX", "T2M_MIN", "T2M", "ALLSKY_SFC_SW_DWN", "RH2M", "T2MDEW", "WS10M"), 
                         dates = c("2018-01-01", paste0(Sys.Date())),
                         temporal_average = "DAILY")
```

**Comment by adamhsparks** [comment]
You could put this at the top of your script.

```r
library(nasapower)
parameters$W2SM$community <- c("AG", "SB", "SSE")
```

That should overwrite the in-package list that says it's not available since they do offer AG and ICASA units, that should be OK until NASA fixes it.

**Comment by danielreispereira** [comment]
Thanks Adam!
I'm using the workaround until the API is fixed. I understand and support the `wont fix` tag.
I will make the `NASA team` aware of this inconsistency.

**Comment by adamhsparks** [comment]
Thanks for your understanding, @danielreispereira 

I'll also contact them since I wrote the package that uses their JSON file.

Soon as NASA is back and they fix the issue (assuming they fix it) I can update it and push to CRAN.

**Comment by danielreispereira** [comment]
Hi Adam, 
See response below from the POWER developer. 
I double-checked and confirm that the APIs links (http) work and the `Rnasapower` translation fails for WS2M.

update:
```
Dear Daniel Pereira,

The POWER developer found that the bug is with your application.

The request based on the data in your Ticket 32:
https://power.larc.nasa.gov/cgi-bin/v1/DataAccess.py?&request=execute&identifier=SinglePoint&parameters=PRECTOT,T2M_MAX,T2M_MIN,T2M,ALLSKY_SFC_SW_DWN,RH2M,T2MDEW,WS10M&startDate=20150101&endDate=20150305&userCommunity=AG&tempAverage=DAILY&outputList=JSON&lat=40.2133&lon=-74.2852

Please let us know if you have additional questions or if you have more information that may be helpful in solving the problem.

Warm Regards,
Michelle
```

```
# this works:
get_power(community = "AG",
                           lonlat = c(-49.554,-20.862, -48.562, -19.851),
                           pars = c("PRECTOT", "T2M_MAX", "T2M_MIN", "T2M", "ALLSKY_SFC_SW_DWN", "RH2M", "T2MDEW", "WS10M"),
                           dates = c("1985-01-01", paste0(Sys.Date())),
                           temporal_average = "DAILY")
# this fails:
get_power(community = "AG",
                           lonlat = c(-49.554,-20.862, -48.562, -19.851),
                           pars = c("PRECTOT", "T2M_MAX", "T2M_MIN", "T2M", "ALLSKY_SFC_SW_DWN", "RH2M", "T2MDEW", "WS2M"),
                           dates = c("1985-01-01", paste0(Sys.Date())),
                           temporal_average = "DAILY")
```

**Comment by adamhsparks** [comment]
Sorry, what? The bug is with the application? Meaning is `nasapower` causing this issue?

I'll have to dig further but the list I use to validate came directly from them.

**Comment by adamhsparks** [comment]
I'm communicating with the POWER team about this. Apparently they didn't intend for this file to actually be used externally. I'm trying to find out from them what the preferred method for validation before sending a request is if not using this file.

**Comment by adamhsparks** [comment]
I've released v1.0.5 with a "fix" for this. The POWER team apparently do not intend for this JSON file to be used externally. However, upon asking there is no clear guidance from the POWER team about how to validate queries before sending them. So I've implemented the "fix", `parameters$W2SM$community <- c("AG", "SB", "SSE")` to apply to the included list used for validation purposes. See https://github.com/ropensci/nasapower/tree/master/data-raw for more.

Hopefully the POWER team can get back to me with a more desirable way of validation if this is not the best way. Until then, this should work.
